specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc

```diff
@@ -594,17 +591,17 @@ link:#a2966[See Dynamic Class Loading]”
 if they need to load classes dynamically.
 
 In addition to the required classes specified
-below, a Java EE product must provide a way to allow an application to
+below, a Jakarta EE product must provide a way to allow an application to
 access a class library installed in the application server, even if it
 has not expressed a dependency on that library. This supports the use of
 old applications and extension libraries that do not use the defined
 extension dependency mechanism.
 
 The following sections describe the
 requirements for each container type. In all cases, access to classes is
-governed by the rules of the Java language and the Java virtual machine.
+governed by the rules of the Jakarta language and the Jakarta virtual machine.
 In all cases, access to classes and resources is governed by the rules
-of the Java security model.
+of the Jakarta security model.
```

Comment:
this should probably remain as Java as well

Reply:
agreed. reverted.